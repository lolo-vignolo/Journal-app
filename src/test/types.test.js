import { types } from "../types/types"

describe('testing types', () => {




    test(' the object should match with the snapshoot ', () => {

        const mytypes = {

            /* authreducer */
            login: '[Auth] Login',
            logout: '[Auth] Logout',
        
            /*uiReducer*/
            uiSetError: "[UI] Set Error",
            uiRemoveError: "[UI] Remove Error",
        
            /**/
            uiStartLoading: '[UI] Start loading',
            uiFinishLoading: '[UI] Finish loading',
        
            /* Work with notes - Sidebar */
        
            noteAddNew : '[Note] New Note',
            noteActive : '[Note] Set active note',
            noteLoad: '[Note] Load Note',
            noteUpDated: '[Note] Note updated',
            noteFileUrl: '[Note] Updated image url',
            noteDelete: '[Note] Delete note',
            noteLogoutCleaning: '[Note] Logout cleaning',
        
        
        }

        expect(types).toEqual(mytypes)

    })
    
    
})
