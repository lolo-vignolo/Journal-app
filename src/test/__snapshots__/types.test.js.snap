// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing types  the object should match with the snapshoot  1`] = `
Object {
  "login": "[Auth] Login",
  "logout": "[Auth] Logout",
  "noteActive": "[Note] Set active note",
  "noteAddNew": "[Note] New Note",
  "noteDelete": "[Note] Delete note",
  "noteFileUrl": "[Note] Updated image url",
  "noteLoad": "[Note] Load Note",
  "noteLogoutCleaning": "[Note] Logout cleaning",
  "noteUpDated": "[Note] Note updated",
  "uiFinishLoading": "[UI] Finish loading",
  "uiRemoveError": "[UI] Remove Error",
  "uiSetError": "[UI] Set Error",
  "uiStartLoading": "[UI] Start loading",
}
`;
